podpac/core/utils.py

```diff
@@ -133,22 +134,14 @@ def optional_import(module_name, package=None, module_attr=None, return_root=Fal
         The imported module if available. None otherwise.
     '''
 
-    try:
-        if return_root:
-            module = importlib.__import__(module_name)
-            if module_attr:
-                raise Exception("Cannot defined 'module_attr' if 'return_root == True'")
-        else:
-            module = importlib.import_module(module_name)
-            if module_attr:
-                module = getattr(module, module_attr)
-    except ImportError:
-        module = None
-    except AttributeError:
-        try: # Python 2.7
-            module = __import__(module_name)
-        except ImportError:
-            module = None
+    if return_root:
+        module = lazy_import.lazy_module(module_name, level='base')
+        if module_attr:
+            raise Exception("Cannot defined 'module_attr' if 'return_root == True'")
+    else:
+        module = lazy_import.lazy_module(module_name)
+        if module_attr:
+            module = getattr(module, module_attr)
```

Comment:
I think this line will actually hard-load the module. Perhaps this requires a little more logic... or perhaps we should scrap the optional_import module completely and just use lazy_import.

Reply:
Let's scrap optional_import and just use lazy_import for all our optional imports.